Integrations/integration-Panorama.yml

```diff
@@ -3357,7 +3352,63 @@ script:
       description: Job ID details
       type: string
     description: Get push status for a configuration
+  - name: panorama-get-pcaps
+    arguments:
+    - name: pcapType
+      required: true
+      auto: PREDEFINED
+      predefined:
+      - application-pcap
+      - filter-pcap
+      - threat-pcap
+      - dlp-pcap
+      description: Type of Packet Capture
+    - name: from
+      description: '"From" value for the rule'
+    - name: to
+      description: '"To" value for the rule'
+    - name: serialNo
+      description: Serial number for the request. For further information, see the
+        Panorama XML API Documentation.
+    - name: searchTime
+      description: Search time for the request. For further information, see the Panorama
+        XML API Documentation.
+    - name: pcapID
+      description: ID of the Pcap for the request. For further information, see the
+        Panorama XML API Documentation.
+    - name: password
+      description: Password for Panorama.
```

Comment:
I think the description is wrong. From a quick search I think this is for dlp-pcaps. Please document when this is needed.